tools/CMakeLists.txt

```diff
@@ -68,9 +68,6 @@ if (NOT ${SPIRV_SKIP_EXECUTABLES})
   endif(SPIRV_BUILD_FUZZER)
 
   if(ENABLE_SPIRV_TOOLS_INSTALL)
-    install(TARGETS ${SPIRV_INSTALL_TARGETS}
-      RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-      LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-      ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+    install(TARGETS ${SPIRV_INSTALL_TARGETS} DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Cmake has evolved since we first wrote this configuration.
If I'm reading the documentation correctly, this is the new recommended way of managing installation destinations.
In particular the project uses GNUInstallDirs (specified in parent directory), which sets RUNTIME, LIBRARY, and ARCHIVE as in the older version of the code here.

I tested an install build before and after this change, on Linux, and the install destinations were the ssame.